pnpm-lock.yaml

```diff
@@ -228,8 +228,8 @@ importers:
       '@lukeed/uuid': 2.0.0
       cookie: 0.4.1
     devDependencies:
-      '@sveltejs/adapter-auto': 1.0.0-next.9
-      '@sveltejs/kit': 1.0.0-next.217_svelte@3.44.2
+      '@sveltejs/adapter-auto': link:../../../adapter-auto
+      '@sveltejs/kit': link:../../../kit
```

Comment:
Pretty sure we don't want this. 

I was hoping leaving these template `package.json`s unignored by renovate would mean they're kept up to date for us but it looks like it is trying to link them to the monorepo (as they are devdeps) but this will break anyone copies one of these templates.

Reply:
I've actually just realised these are `link` not `workspace`. I need to look into this but I'll probably just ignore the templates for now.